pkgs/by-name/pi/picom-pijulius/package.nix

```
@@ -0,0 +1,27 @@
+{
+  picom,
+  lib,
+  fetchFromGitHub,
+  pcre
+}:
+picom.overrideAttrs (
+  finalAttrs: previousAttrs: {
+    pname = "picom-pijulius";
+    version = "8.2-2024-05-17-unstable";
```

```suggestion
    version = "8.2-unstable-2024-04-30";
```
This needs to be the date of the commit, not today's date.

Also, please change the PR title and the commit message to say `picom-pijulius: init at 8.2-unstable-2024-04-30`